packages/block-library/src/image/image-editing/cropper.js

```diff
@@ -51,6 +52,10 @@ export default function ImageCropper( {
 				width: width || clientWidth,
 				height: editedHeight,
 			} }
+			onClick={ onClick }
+			onKeyDown={ onClick }
```

Comment:
I'm not entirely sure how accessible this is via keyboard anyway.

Are folks happy with adding a 
```
// eslint-disable-next-line jsx-a11y/no-static-element-interactions

```

to disable for now?

Reply:
I don't have any objections to disabling the `no-static-element-interactions` linting for this. It appears to be done in a dozen or more places already.

The accessibility concerns here aren't my strong suit so happy to defer to others there.